build_datacard.py

```diff
@@ -135,25 +137,33 @@ def apply_selection(cols,year):
         elif year=="2018POST":
             if metadata["sample_type"]=="data":
                 cols = cols.select(cols.arrays['run']>=startHEM)
-            cols = common.apply_hemveto(cols)
+            if selection != "preselection_minus":
```

Comment:
this change should not be necessary, because `2018POST` is a selection used to compare with data. N-1 plots are MC-only, so they will only use the generic `2018` selection.

Reply:
@yimuchen this is still unresolved

Reply:
Ah, I see that is going on. In this case, should I update the instructions in the "README" to add the `--full_year` flag for the build histogram instructions for the signal and background MC samples? 

https://github.com/boostedsvj/svj_uboost/blob/main/README.md?plain=1#L160

Reply:
No, the instructions are correct. For data/MC comparisons, we have to apply the same selections to both data and MC, so the HEM veto is used on all samples. The N-1 plots are not data/MC comparisons; they are MC-only, so the HEM veto is not needed.

Reply:
Ah, I see. So I guess I would update the instructions in the svj_paperplots repository in this case. And this issue can be marked as resolved?